senders/solomon/solomon.go

```diff
@@ -2,11 +2,10 @@ package solomon
 
 import (
 	"bytes"
+	"context"
```

Comment:
as far as I know ctxhttp returns errors from non-standard context pakage, doesn't it?